python/sglang/srt/layers/decode_attention.py

```diff
@@ -214,14 +196,7 @@ def _decode_att_m_fwd(
     # shape constraints
     Lq, Lk = q.shape[-1], k_buffer.shape[-1]
     assert Lq == Lk
-    assert Lk in {16, 32, 64, 128, 256, 576}
-
-    if Lk == 576:
-        BLOCK_DMODEL = 512
-        BLOCK_DPE = 64
-    else:
-        BLOCK_DMODEL = Lk
-        BLOCK_DPE = 0
+    assert Lk in {16, 32, 64, 128, 256}
```

Comment:
Why remove 576?

Reply:
We will call `_fwd_kernel_stage1` for MHA and  `_fwd_grouped_kernel_stage1`  for MLA.

Reply:
Ok. It's for MHA before. Ignore it.